tests/op/extract/load/test_txt_op.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from unittest.mock import patch
+
+from uniflow.node import Node
+from uniflow.op.extract.load.txt_op import ExtractTxtOp, ProcessTxtOp
+
+
+class TestExtractTxtOp(unittest.TestCase):
+    def setUp(self):
+        self.extract_txt_op = ExtractTxtOp(name="extract_txt_op")
+
+    @patch(
+        "uniflow.op.extract.load.txt_op.read_file", return_value="mocked file content"
+    )
+    def test_call(self, mock_read_file):
+        # arrange
+        node = Node(name="node1", value_dict={"filename": "mocked_file_path"})
```

Comment:
nit: it might worth to add two more unittest to test with `more than one node` and `no node.` You should aggregate all into a single test_call through parameterized unit testing. https://medium.com/@samarthgvasist/parameterized-unit-testing-in-python-9be82fa7e17f

Reply:
done